aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -138,4 +142,84 @@ void RegisterOperators::registerOp_(Options&& options) {
   }
 }
 
+// This map is used to enable users to use PYTORCH_CPU_FALLBACK_OPS=all. There are
+// some ops that cannot fallback to CPU, these ops are listed here to prevent cases
+// where such ops are un-registered.
+// TODO: Find programmatic way to generate this list
+static std::unordered_set<std::string> mustRegisterOpNames{
```

Comment:
Could we have this list where the cpu fallback is registered and just register the fallback as impl there to override the ones from the codegen?